Tools/Scripts/webkitdirs.pm

```diff
@@ -3352,12 +3352,13 @@ sub runIOSWebKitApp($)
     die "Not using an iOS SDK."
 }
 
-sub archCommandLineArgumentsForRestrictedEnvironmentVariables()
+sub commandLineArgumentsForRestrictedEnvironmentVariables($)
 {
+    my ($prefix) = @_;
     my @arguments = ();
     foreach my $key (keys(%ENV)) {
         if ($key =~ /^DYLD_/) {
-            push @arguments, "-e", "$key=$ENV{$key}";
+            push @arguments, $prefix, "$key=$ENV{$key}";
```

Comment:
This change is also perfectly fine, although I'd probably rewrite this with a combination of `grep` and `map`, just to show off :-)